packages/react/src/primitives/SearchField/SearchFieldSuggestionMenu.tsx

```diff
@@ -0,0 +1,224 @@
+          <Loader />
+          Loading suggestions...
+        </Flex>
+      ),
+    [Loading]
+  );
+
+  const NoSuggestions = React.useCallback(
+    () =>
+      Empty ? (
+        <Flex className={ComponentClassNames.SearchFieldMenuEmpty}>
+          {Empty}
+        </Flex>
+      ) : (
+        <Flex className={ComponentClassNames.SearchFieldMenuEmpty}>
+          No suggestions found
+        </Flex>
+      ),
+    [Empty]
+  );
+
+  const Suggestions = React.useCallback(() => {
+    return suggestions.length > 0 ? (
+      <ScrollView
+        as="ul"
+        role="listbox"
+        ariaLabel={ariaLabel}
+        className={ComponentClassNames.SearchFieldMenuSuggestions}
+        id={id}
+      >
+        {suggestions.map((suggestion: Suggestion, idx) => {
+          const { id, label } = suggestion;
+          const isActive = activeIdx === idx;
+
+          const handleOnClick: React.MouseEventHandler<HTMLLIElement> = () => {
+            setIsMenuOpen(false);
+            setActiveIdx(-1);
+            if (!isControlled) {
+              setInternalValue(label);
+            }
+            if (isFunction(onSuggestionSelect)) {
+              onSuggestionSelect(suggestion);
+            }
+          };
+
+          const handleOnMouseDown: React.MouseEventHandler<HTMLLIElement> = (
+            event
+          ) => {
+            event.preventDefault();
+          };
+
+          const handleOnMouseMove: React.MouseEventHandler<HTMLLIElement> =
+            () => {
+              setActiveIdx(idx);
+            };
+
+          const suggestionId = `${suggestionBaseId}-suggestion-${idx}`;
+
+          return (
+            <SearchFieldSuggestion
+              aria-selected={isActive}
+              className={classNameModifierByFlag(
+                ComponentClassNames.SearchFieldMenuSuggestion,
+                'active',
+                isActive
+              )}
+              id={id || suggestionId}
+              key={id || suggestionId}
+              onClick={handleOnClick}
+              onMouseDown={handleOnMouseDown}
+              onMouseMove={handleOnMouseMove}
+            >
+              {isFunction(renderSuggestion) ? (
+                renderSuggestion(suggestion, value)
+              ) : filteringType !== 'manual' ? (
+                <HighlightMatch query={value}>{label}</HighlightMatch>
+              ) : (
+                label
+              )}
+            </SearchFieldSuggestion>
+          );
+        })}
+      </ScrollView>
+    ) : (
+      <NoSuggestions />
+    );
+  }, [
+    NoSuggestions,
+    activeIdx,
+    ariaLabel,
+    filteringType,
+    id,
+    isControlled,
+    onSuggestionSelect,
+    renderSuggestion,
+    setActiveIdx,
+    setInternalValue,
+    setIsMenuOpen,
+    suggestionBaseId,
+    suggestions,
+    value,
+  ]);
+
+  React.useEffect(() => {
+    const menuContainerElement = document.getElementById(menuContainerId);
```

Comment:
This will improve the user experience. The window will scroll down to a place where the whole dropdown is displayed if the space is not enough in current viewport.

![autocomplete_scroll](https://user-images.githubusercontent.com/40295569/194903249-7b2164b6-6c7e-4e00-b97f-fca55558be2d.gif)